pkg/workload/cli/run.go

```diff
@@ -428,10 +435,65 @@ func runRun(gen workload.Generator, urls []string, dbName string) error {
 	if *individualOperationReceiverAddr != "" {
 		publisher = histogram.CreateUdpPublisher(*individualOperationReceiverAddr)
 	}
-	reg := histogram.NewRegistryWithPublisher(
+
+	var exporter histogram.Exporter
```

Comment:
Can we move it to a separate function which returns the exporter?

Reply:
Done